### PR TITLE
Added check, whether a user is still in the guild. 

### PR DESCRIPTION
**Pull request by Shirkanesi**

### CODE REVIEW

**Review by yJulian — APPROVED** [review]
Looks okay

**Review by yNiklas — COMMENTED** [review]
Warum junge

File: src/main/java/tech/ypsilon/bbbot/discord/DiscordController.java [hunk 1]
```diff
@@ -23,8 +23,9 @@
     public DiscordController() throws LoginException {
         instance = this;
         Collection<GatewayIntent> gatewayIntents = Arrays.asList(GatewayIntent.GUILD_VOICE_STATES,
-                GatewayIntent.GUILD_MESSAGES, GatewayIntent.GUILD_MESSAGE_REACTIONS, GatewayIntent.DIRECT_MESSAGES,
-                GatewayIntent.GUILD_MEMBERS);
+                GatewayIntent.GUILD_MESSAGES, GatewayIntent.GUILD_MESSAGE_REACTIONS, GatewayIntent.DIRECT_MESSAGES
+                , GatewayIntent.GUILD_MEMBERS
```
Comment (Gregyyy) [inline src/main/java/tech/ypsilon/bbbot/discord/DiscordController.java:27]: Why was this line changed in this way?

File: src/main/java/tech/ypsilon/bbbot/discord/command/BirthdayCommand.java [hunk 2]
```diff
@@ -1,390 +1,404 @@
...
+     * @param userId
+     * @param bday
+     * @param guild
+     * @param channel
+     * @return
+     */
+    public static boolean shoutOutBday(long userId, Date bday, Guild guild, MessageChannel channel) {
+        if (hasBirthdayToday(bday)) {
+            User user = guild.getJDA().retrieveUserById(userId).complete();
+            if (guild.isMember(user)) {
```
Comment (Gregyyy) [inline src/main/java/tech/ypsilon/bbbot/discord/command/BirthdayCommand.java:266]: This will not work. Guild#isMember only checks the member cache. Since the MemberCachePolicy was not specifically set when the Discord bot was build, JDA uses DEFAULT. This will result in only members in voice channels and the owner being cached.

**Review by Gregyyy — COMMENTED** [review]
The current Implementation will not work, because of MemberCachePolicy. More details in line commands.

File: src/main/java/tech/ypsilon/bbbot/discord/DiscordController.java (hunk 1 above)
Comment (Gregyyy) [reply src/main/java/tech/ypsilon/bbbot/discord/DiscordController.java:27]: Maybe we should add a better debug mode
